upup/pkg/fi/cloudup/terraform/element.go

```diff
@@ -71,24 +72,81 @@ func (o *object) Write(buffer *bytes.Buffer, indent int, key string) {
 			maxKeyLen = 0
 		}
 		o.field[key].Write(buffer, indent+2, key)
-		buffer.WriteString("\n")
 	}
 	writeIndent(buffer, indent)
-	buffer.WriteString("}")
+	buffer.WriteString("}\n")
 }
 
+type sliceObject struct {
+	members []element
+}
+
+func (s *sliceObject) IsSingleValue() bool {
+	return false
+}
+
+func (s *sliceObject) Write(buffer *bytes.Buffer, indent int, key string) {
+	for _, member := range s.members {
+		member.Write(buffer, indent, key)
```

Comment:
I was surprised we didn't need `[` or `,` here.  But now I see that it's because these are slices of objects, which are just written without delimeters.  Makes sense, but a comment might be helpful here.